### ISSUE TITLE
Gives deprecation warning on rails 5

### ISSUE THREAD

**Issue by lephyrius** [pboling/stackable_flash#5]
```
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from <top (required)> at 
/usr/local/var/rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/stackable_flash-0.0.7/lib/stackable_flash/stack_layer.rb:7:in `block in included'
```

This is what I get when I try to use it under rails 5.

**Comment by pboling** [comment]
Thanks for the report.  This gem definitely needs some love.  I haven't had an opportunity to use it in some years.  Making this change would be something of a re-write, but a much needed one.

**Comment by fredngo** [comment]
Hi @pboling, I've submitted a PR which solves this issue: https://github.com/pboling/stackable_flash/pull/7

**Comment by pboling** [comment]
Awesome!  Thanks!

**Comment by yuki24** [comment]
I believe this issue has been resolved thanks to @fredngo for fixing it. @pboling Could you cut a new release?

**Comment by pboling** [comment]
@yuki24 @fredngo - released v0.1.1 with rails 5 support.

**Comment by yuki24** [comment]
@pboling Thank you for the new release! really appreciate it.
